oaknational/Oak-Web-Application
# [CUR-185] Added curriculum info overview tabs

## Description

- Added further slug for redirects
- Added base components for overview and sequence


## How to test

1. Go to https://deploy-preview-1780--oak-web-application.netlify.thenational.academy/beta/teachers/curriculum/maths-secondary/overview
2. You should see the curriculum overview for secondary maths
3. Click on the sequence tabular nav
4. You should see a page reflecting the subject phase sequence
5. Click on the download tab
6. You should see a page reflecting the download section for the curricula
7. Test the breadcrumbing redirects properly between subject and phase

## Checklist
